tests/unit/BusinessContactInfo.spec.ts

```diff
@@ -38,6 +38,10 @@ describe('BusinessContactInfo for a Correction', () => {
     }
   })
 
+  afterAll(() => {
+    store.state.stateModel.correctedFiling = null
```

Comment:
State is global (within a test suite/file) so need to reset it for other tests to run correctly.